fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java

```diff
@@ -301,13 +301,33 @@ public static void estimate(GroupExpression groupExpression, CascadesContext con
 
     private void estimate() {
         Plan plan = groupExpression.getPlan();
-        Statistics newStats = plan.accept(this, null);
+        Statistics newStats;
+        try {
+            newStats = plan.accept(this, null);
+        } catch (Exception e) {
+            // throw exception in debug mode
+            if (ConnectContext.get() != null && ConnectContext.get().getSessionVariable().feDebug) {
+                throw e;
+            }
+            LOG.warn("stats calculation failed, plan " + plan.treeString(), e);
```

Comment:
why need to print all plan tree? it is very expensive

Reply:
The reason for printing the plan tree is based on the following considerations:
1. The plan tree is only printed when there is an error, and errors rarely occur here.
2. If we have a complete plan tree, it will be convenient for us to locate the problem more quickly, without the need to execute “explain physical plan” again, especially in POC.